pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -12,16 +12,16 @@ import (
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
-// VerifyZero hint verifies that a packed value is zero modulo the SECP256R1 prime
+// VerifyZero hint verifies that a packed value is zero modulo the SECP prime
```

Comment:
can you replace it with `secp256k1` ? 
just chcked and this hint is related to `secp256k1` curve

Reply:
done.